### DISCUSSION TITLE
Thanks for the Pants Examples

### DISCUSSION THREAD

**Discussion by ryaminal** [lablup/backend.ai#1494] [General]
Hi backend.ai,

Not only does backend.ai look like a super useful tool for me, at the moment, but I've also started exploring using pantsbuild for a lot of our stuff. This repo has been a huge help in standing up pants as it's a real world, "complex" solution that was easy to follow.

Just thought I'd share my gratitude for this!

**Comment by achimnol** [comment]
This is a bit late reply, but thanks! And also kudos to the pantsbuild team as well for making such a great tool!
